src/Teams/Models/TeamMember.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Teams.Models
+{
+    public class TeamMember
+    {
+        [Column("Id", TypeName = "int")]
+        [Required(AllowEmptyStrings = false)]
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+        [Column("TeamId", TypeName = "int")]
```

Comment:
this should look like
`[ForeignKey(nameof(Team)), Required]
public int TeamId { get; set; }
public virtual Team Team { get; set; }`